vagrant/Vagrantfile

```diff
@@ -0,0 +1,26 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+
+    # Ubuntu 16.04 LTS
+    config.vm.box = "fosskers/docker-1.12"
```

Comment:
I think that this approach can easily turn into a slippery slope. Right now, `fosskers/docker-1.12` is a bit of a black box.

- What process was used to create it?
- How can someone else update it? Report issues?
- What happens when the Docker Engine needs to be updated? What about Compose?

I'm not suggesting that the approach with Ansible is the ideal, but with that approach, all of these maintenance related questions have proven answers.